src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -529,7 +529,7 @@ public virtual void Sync()
             DateTime start, stop;
             start = DateTime.UtcNow;
             // REMOTING CALL
-            SyncedMessages = _ChatModel.Messages;
+            SyncedMessages = _ChatModel.MessageBuffer;
```

Comment:
Actually even in master MessageBuffer is not [Serializable], see https://github.com/meebey/smuxi/blob/master/src/Engine/MessageBuffers/SqliteMessageBuffer.cs#L31 but in the sync_tail branch, which isn't merged since it past the code/feature freeze for the next release.